tests/e2e/compare/output/markdown.ts

```diff
@@ -1,80 +1,80 @@
 // From: https://raw.githubusercontent.com/callstack/reassure/main/packages/reassure-compare/src/output/markdown.ts
 import fs from 'node:fs/promises';
 import path from 'path';
-import _ from 'underscore';
+import type {Stats} from 'tests/e2e/measure/math';
 import * as Logger from '../../utils/logger';
+import type {Data, Entry} from './console';
 import * as format from './format';
 import markdownTable from './markdownTable';
 
 const tableHeader = ['Name', 'Duration'];
 
-const collapsibleSection = (title, content) => `<details>\n<summary>${title}</summary>\n\n${content}\n</details>\n\n`;
+const collapsibleSection = (title: string, content: string) => `<details>\n<summary>${title}</summary>\n\n${content}\n</details>\n\n`;
 
-const buildDurationDetails = (title, entry) => {
+const buildDurationDetails = (title: string, entry: Stats) => {
     const relativeStdev = entry.stdev / entry.mean;
 
-    return _.filter(
-        [
-            `**${title}**`,
-            `Mean: ${format.formatDuration(entry.mean)}`,
-            `Stdev: ${format.formatDuration(entry.stdev)} (${format.formatPercent(relativeStdev)})`,
-            entry.entries ? `Runs: ${entry.entries.join(' ')}` : '',
-        ],
-        Boolean,
-    ).join('<br/>');
+    return [
+        `**${title}**`,
+        `Mean: ${format.formatDuration(entry.mean)}`,
+        `Stdev: ${format.formatDuration(entry.stdev)} (${format.formatPercent(relativeStdev)})`,
+        entry.entries ? `Runs: ${entry.entries.join(' ')}` : '',
+    ]
+        .filter(Boolean)
+        .join('<br/>');
 };
 
-const buildDurationDetailsEntry = (entry) =>
-    _.filter(['baseline' in entry ? buildDurationDetails('Baseline', entry.baseline) : '', 'current' in entry ? buildDurationDetails('Current', entry.current) : ''], Boolean).join(
-        '<br/><br/>',
-    );
+const buildDurationDetailsEntry = (entry: Entry) =>
+    ['baseline' in entry ? buildDurationDetails('Baseline', entry.baseline) : '', 'current' in entry ? buildDurationDetails('Current', entry.current) : '']
+        .filter(Boolean)
+        .join('<br/><br/>');
 
-const formatEntryDuration = (entry) => {
-    if ('baseline' in entry && 'current' in entry) {
+const formatEntryDuration = (entry: Entry) => {
+    if (entry.baseline && entry.current) {
         return format.formatDurationDiffChange(entry);
     }
-    if ('baseline' in entry) {
+    if (entry.baseline) {
         return format.formatDuration(entry.baseline.mean);
     }
-    if ('current' in entry) {
+    if (entry.current) {
```

Comment:
@blazejkustra @fabioh8010 

Good point! I think that these checks don't make sense because `baseline` and `current` should be always present according to the code, but could you both check this? I need more pair of eyes here because I'm not able to test this code

Reply:
If we can;t test it, the safest way would be to leave the logic as it was, wdyt?

Reply:
@blazejkustra reverting the logic throws typescript errors

<img width="954" alt="Screenshot 2024-03-22 at 15 27 41" src="https://github.com/Expensify/App/assets/48553277/5ac604df-0800-4ff6-90ef-e05df9a5a1a4">

Reply:
@pac-guerreiro This way will work:

```ts
const formatEntryDuration = (entry: Entry): string => {
    let formattedDuration = '';

    if ('baseline' in entry && 'current' in entry) {
        formattedDuration = format.formatDurationDiffChange(entry);
    }

    if ('baseline' in entry) {
        formattedDuration = format.formatDuration(entry.baseline.mean);
    }

    if ('current' in entry) {
        formattedDuration = format.formatDuration(entry.current.mean);
    }

    return formattedDuration;
};
```

Reply:
@fabioh8010 thanks, this helped 😄